memberships/static/memberships/js/main.js

```diff
@@ -0,0 +1,33 @@
+const hello = () => console.log("hello");
+var header;
+
+// if scrolled down, make header sticky
+function checkSticky() {
```

Comment:
Apparently it is possible to do a sticky header entirely in CSS and the support is pretty good.

https://w3codepen.com/howto-html-css-fixed-sticky-header/
https://caniuse.com/?search=sticky

Reply:
That's a heck of a lot simpler - only needs three lines of CSS apparently.